cypress/integration/chi-js/modal.spec.js

```diff
@@ -0,0 +1,182 @@
+describe('chi-modal', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/modal.html');
+  });
+
+  describe('check base modal', () => {
+    it(`modal should be opened when we click launch demo modal button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .should('have.class', '-active')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+
+    it(`modal should be closed when we click close button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .find('.-close')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
+
+    it(`modal should be closed when we click cancel button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .contains('Cancel')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
+  });
+
+  describe('check Center modal', () => {
+    it(`modal should be open in the center of the page when we click launch demo modal button`, () => {
+      cy.get('#modal-trigger-2').click();
+      cy.get('#modal-2')
+        .parent()
+        .parent()
+        .should('have.class', '-center')
+        .find('.-close')
+        .wait(550)
+        .click();
+    });
+  });
+
+  describe('check scrollable modal', () => {
+    it(`modal should be scrollable`, () => {
+      cy.get('button#modal-trigger-3')
+        .click()
+        .wait(550);
+      cy.get('#modal-3 .chi-modal__content')
+        .should('have.class', '-mh--400')
```

Comment:
The class is provided in HTML markup.
The test will always give false positive since the context of the test is supposed to be checking if the Modal has a scrollbar.
Please check if Modal content overflow-y property is set to auto
`overflow-y: auto;`
Since this is the property that provides scrolling funtionality

Reply:
Fixed in the latest commit